subprojects/docs/src/snippets/java/apt/kotlin/build.gradle.kts

```diff
@@ -9,9 +9,9 @@ repositories {
 // tag::annotation-processing[]
 dependencies {
     // The dagger compiler and its transitive dependencies will only be found on annotation processing classpath
-    annotationProcessor("com.google.dagger:dagger-compiler:2.8")
+    annotationProcessor("com.google.dagger:dagger-compiler:2.44")
 
     // And we still need the Dagger library on the compile classpath itself
-    implementation("com.google.dagger:dagger:2.8")
+    implementation("com.google.dagger:dagger:2.44")
```

Comment:
BTW why do we have to decrease the version?

Reply:
It is an increase, `8 < 44` :). Dagger 2.8 is not compatible with Java 11, so I had to bump the version.

Reply:
Ah I see, probably too late for review :)